static/ng/resources/js/layout/project/project.controller.js

```diff
@@ -11,6 +11,13 @@
   function ProjectController($scope, ListProjectService, $timeout, currentUser) {
     var vm = this;
     
+    vm.MAP = {
```

Comment:
We've already created a factory service named 'getRole' in the /static/ng/js/components/project-member.config.js, however, it may need to move upward to fit other modules to inject it.